.gitlab-ci/stages/deploy.yml

```diff
@@ -122,26 +120,11 @@ Deploy @cdtn/frontend (dev):
   extends:
     - .deploy_frontend_stage
     - .dev_stage
-  dependencies:
-    - Register nlp image
   variables:
     HELM_RENDER_ARGS: >-
       --set autoscaling.enabled=false
       --set deployment.resources.requests.memory=256Mi
       --set deployment.resources.requests.cpu=.1m
-  artifacts:
-    expire_in: 30 mins
-    paths:
-      - packages/code-du-travail-nlp
-  after_script:
-    - |-
-      source ./.gitlab-ci/env.sh
-      retry=60;
-      while
-        ! curl -sS "${FRONTEND_URL}/api/version" | grep "${COMMIT}" &&
-        [[ $(( retry-- )) -gt 0 ]];
-      do echo "Waiting for frontend to be ready" ; sleep 3 ; done ;
-      [ "$retry" -eq "-1" ] && exit 1
```

Comment:
this feels of topic of the pr non ?
could it be merged in a separate pr ?

Reply:
once we will a robust ci, I will do micro PR